designs/administrator/olm/installing-operator-long-running-component/index.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Operator Installing with Common Long Running Styling
+
+Shortly after the new screen to [Convey an operator is installing](http://openshift.github.io/openshift-origin-design/designs/administrator/olm/convey-installing-operator/) was developed, it was realized that other actions in the console could make use of a similar interaction. This design conveys a more generic version of the operator installing screen, and examples of perhaps how other actions could use a similar interaction and styling.
+
+## Operator Installation
+
+These screens show how the existing Installing Operator feature could appear using newer, more generic, visuals, which could perhaps be reused for other use cases.
+
+### Installing an operator
+![Install operator](img/longRun-1-1-installing.png)
+
+![Installed operator](img/longRun-1-2-installed.png)
+
+### Installing an operator with required custom resource
+
+![Install operator CR](img/longRun-1-3-installingCR.png)
+
+![Installed operator CR](img/longRun-1-4-installedCR.png)
+
+### Manual approval required
+
+![Install operator manual approval](img/longRun-1-5-manApprove.png)
```

Comment:
Again, not sure this matters for this wireframe, so you'll have to let me know, but ideally the text would have a break on this screen and wouldn't span across the full width.

Reply:
That makes sense to me, just updated the image, thanks!